### ISSUE TITLE
Empirical Performance Analysis of Conventional Deep Learning Models for Recognition of Objects in 2-D Images. (arXiv:2011.06639v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#1011]
Empirical Performance Analysis of Conventional Deep Learning Models for Recognition of Objects in 2-D Images. (arXiv:2011.06639v1 [cs.CV])<br>
https://ift.tt/3lExNgQ<br>
<p>Artificial Neural Networks, an essential part of Deep Learning, are derived from the structure and functionality of the human brain. It has a broad range of applications ranging from medical analysis to automated driving. Over the past few years, deep learning techniques have improved drastically - models can now be customized to a much greater extent by varying the network architecture, network parameters, among others. We have varied parameters like learning rate, filter size, the number of hidden layers, stride size and the activation function among others to analyze the performance of the model and thus produce a model with the highest performance. The model classifies images into 3 categories, namely, cars, faces and aeroplanes.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
